src/screen.c

```diff
@@ -38,7 +38,7 @@ slider_location_t bottom_slider = 0;
 void screen_print_debug(const char* message, int duration)
 {
     char print[100];
-    strncpy(print, message, sizeof(print));
+    strncpy(print, message, sizeof(print) - 1);
     print[sizeof(print) - 1] = 0;
```

Comment:
could use `snprintf` to avoid manually handling the terminator?

Reply:
ack